src/css/components/_keyboard.scss

```diff
@@ -4,7 +4,29 @@
     display: flex;
     justify-content: center;
     flex-direction: column;
-    padding: 20px;    
+    padding: 20px;
+    padding-top: 0px;
+}
+
+.keyboard .vscrollmenu {
+    height: 40%;
+}
+
+.keyboard .hscrollmenu {
+    padding-top: 10px;
+    height: 40%;
+    display: flex;
+    justify-content: space-between;
```

Comment:
Might have to limit the height of the hscrollitem primary image as well
![Screenshot from 2022-02-02 15-46-07](https://user-images.githubusercontent.com/10703941/152357516-e9ca3baa-0466-48e4-b385-a162355ff1f4.png)

Reply:
The primary image looks good with the changes but might have to apply similar constraints to the secondary image
![Screenshot from 2022-02-07 09-56-17](https://user-images.githubusercontent.com/10703941/152813471-ec1ea585-0b26-4bac-83df-989c1d756796.png)